https://issues.redhat.com/browse/ACM-8261#links, versions, edits across docs (#5684)
open-cluster-management/rhacm-docs: install/cluster_size.adoc

| 4
| 16 GB
| 120 GB

> | See the link:https://docs.openshift.com/container-platform/4.12/installing/installing_aws/installing-aws-customizations.html#installing-aws-customizations[Amazon Web Services information in the {ocp-short} product documentation] for more information.

Reviewer: ```suggestion
| See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/installing/installing-on-aws#installing-aws-customizations[Installing a cluster on AWS with customizations] in the {ocp-short} product documentation for more information.
```

| 4
| 16 GB
| 120 GB

> | See the following link:https://docs.openshift.com/container-platform/4.12/installing/installing_azure/installing-azure-account.html[product documentation] for more details.

Reviewer: ```suggestion
| See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/installing/installing-on-azure#installing-azure-account[Configuring an Azure account] in the {ocp-short} documentation for more details.
```
